model/src/main/proto/interaction_object.proto

```diff
@@ -95,6 +95,8 @@ message ImageWithRegions {
     }
 
     Region region = 2;
+    // The description for the region.
```

Comment:
Please be specific: what kind of description? How is this used by the application?

Reply:
done